### PR TITLE
Ce 2187 tcs stats

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Added methods to get data about number of classified templates:

- ```getClassifiedTemplatesCount``` - returns number of all classified templates from all wikis
- ```getClassifiedTemplatesOnWikiCount``` - returns number of classified templates from one wiki
- ```getClassifiedTemplatesByProviderCount``` - returns number of all classified templates by given provider from all wikis
- ```getClassifiedTemplatesByProviderOnWikiCount``` - returns number of classified templates by given provider from one wiki

ping @sqreek

### CODE REVIEW

File: service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/resources/TemplateClassificationStatsResource.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
+package com.wikia.services.templateclassificationstorage.resources;
+
+import com.wikia.services.templateclassificationstorage.dao.ITemplateClassificationStatsStorage;
+import com.wikia.services.templateclassificationstorage.domain.TemplateClassificationData;
```
Comment (WikiaTech) [inline service/template-classification-storage/src/main/java/com/wikia/services/templateclassificationstorage/resources/TemplateClassificationStatsResource.java]: MINOR Remove this unused import 'com.wikia.services.templateclassificationstorage.domain.TemplateClassificationData'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [6 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/49b785dff7d2ba3bebf8f63b8eb2e2aafc4b899b/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/49b785dff7d2ba3bebf8f63b8eb2e2aafc4b899b/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [6 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/7bb215863c93e47949437119e13617e70fd4fcfb/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/7bb215863c93e47949437119e13617e70fd4fcfb/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/1f9eefc8c8077f1974a389722f96f7feae4cabcf/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/1f9eefc8c8077f1974a389722f96f7feae4cabcf/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/77f56e6faf684ab3ab4409c986c464e008917008/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/77f56e6faf684ab3ab4409c986c464e008917008/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/ed0b97b25485c0eff85f37f5816a1c38d28de335/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/ed0b97b25485c0eff85f37f5816a1c38d28de335/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

File: service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java [hunk 2]
```diff
@@ -50,26 +57,19 @@
...
-  public static final String AUTO_PROVIDER_VERSION = "1.0";
-  private static final int DB_TIMEOUT = 60;
-  private static final Integer WIKI_ID = 100;
-  private static final Integer WIKI_OTHER_ID = 101;
-  private static final Integer PAGE_ID = 200;
+
   private static final String URL = "http://localhost:%d/%d";
   private static final String URL_PAGE = String.format("%s/%%d", URL);
-  private static final String URL_PROVIDERS = String.format("%s/providers", URL_PAGE);
+  private static final String URL_PROVIDERS = String.format("%s/%%d/providers", URL);
```
Comment (sqreek) [inline service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java]: Please remove pattern that using string format in string format.

**Comment by sqreek** [comment]
Except bad pattern that using string format into string format into string format rest is :+1: .

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/70be473b0a8f6a8a3ea39324961f84b7a301fa3e/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStatsResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* MAJOR [1 duplicated blocks of code must be removed.](https://github.com/Wikia/pandora/blob/70be473b0a8f6a8a3ea39324961f84b7a301fa3e/service/template-classification-storage/src/test/java/com/wikia/services/templateclassificationstorage/integration/TemplateClassificationStorageResourceIntegrationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)

**Comment by sqreek** [comment]
:+1: 
LGTM
Please only reformat places that we are talking about offline, and merge it

**Pull request merged**